### ISSUE TITLE
Pythin example doesn't show anything in the simulation

### ISSUE THREAD

**Issue by MattiP99** [raisimTech/raisimLib#552]
Hi,
After the installation of raisim I wanted to test it and see if everything was properly set up.
I launched the raisimUnity environment and then I tried the python examples. I expected something to be shown inside the environment but there was nothing.
I changed the python path which the application was pointing to because there was an error with "import raisimPy" but then the visual studio terminal doesn´t show me any other errors.  
Do you know what could be the problem or how could I debug my code?
Thanks in advance

**Comment by MattiP99** [comment]
In the VS terminal I can just see the following:
![Screenshot from 2024-02-14 14-49-11](https://github.com/raisimTech/raisimLib/assets/92534255/c723347f-93ae-40a3-b6d0-21e2ce34b330)

and nothing happens

**Comment by jhwangbo** [comment]
Did you open the visualizer? Raisim alone cannot visualize the simulation

**Comment by MattiP99** [comment]
Do you mean RaisimUnity or a specific visualizer in Visual Studio? 
Because I've firstly started RaisimUnity and only then I've tried to run the python exercises. 
If it is required a visualizer in VS I didn't know . Do I have to install it or is it already integrated in VS and should I somehow activate it?
Sorry for the trivial questions, I've really appreciated your help.
Thanks again

**Comment by jhwangbo** [comment]
there is an executable in the raisimUnity folder. Run it and you will see what I mean

Please read the visualizer section in the manual as well

**Comment by MattiP99** [comment]
Yes I've already did that . Do you mean the file raisimUnity.x86_64 inside linux folder inside raisimUnity folder?
After launching that an empty environment appears and I was trying to have something shown inside by launching a python example but nothing happens

**Comment by jhwangbo** [comment]
Did you check 'auto-connect'?

**Comment by MattiP99** [comment]
Ah, I didn't check the auto connect label. Now that I did that, It seems to work properly. There is the robot shown in the environment. Thanks very much.
